templates/hooks/rule/sdk_file_end.go.tpl

```diff
@@ -0,0 +1,27 @@
+func SdkTargetsFromResourceTargets(
```

Comment:
Consider adding quick docstrings for these two functions.

Reply:
👍